addons/portal_rating/models/mail_message.py

```diff
@@ -40,12 +40,15 @@ def _portal_message_format(self, properties_names, options=None):
             for rating in related_rating
         }
 
+        rating_stats = {}
+        if self:
+            record = self.env[self[0].model].browse(self[0].res_id)
+            if hasattr(record, 'rating_get_stats'):
+                rating_stats = record.sudo().rating_get_stats()
```

Comment:
Would say we do this properly by grouping models and res_id, it works now with multiple messages and we shouldn't break that in stable.

Something of this order
```py
        rating_stats_by_model = {
            model: {
                res_id: self.env[model].browse(res_id).rating_get_stats()
                if res_id and hasattr(self.env[model], 'rating_get_stats') else None
                for res_id, _ in messages.grouped('res_id')
            } for model, messages in self.grouped('model') if model
        }
```
which should collapse to a single call in most cases but still support cases with messages from multiple records